name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecurityDetailsViewer.java

```diff
@@ -364,13 +367,22 @@ public SecurityDetailsViewer(Composite parent, int style, Client client)
 
     public SecurityDetailsViewer(Composite parent, int style, Client client, boolean showMasterData)
     {
-        container = new Composite(parent, style);
-        container.setBackground(Colors.WHITE);
-        container.setBackgroundMode(SWT.INHERIT_FORCE);
+        container = new ScrolledComposite(parent, SWT.V_SCROLL);
+
+        Composite container1 = new Composite(container, style);
+        container.setContent(container1);
+        container.setExpandHorizontal(true);
+        container.setExpandVertical(true);
+        // Needed to avoid artifact (at least on Linux) when scrollbar appears
+        // and covers right edge (last digits of quotes, etc.) of the content
+        // pane. With setAlwaysShowScrollBars(true) that issue doesn't happen.
+        container.setAlwaysShowScrollBars(true);
+        container1.setBackground(Colors.WHITE);
```

Comment:
I am unsure if this will work with PP's Dark or Withe themes.

Reply:
Tested now on Linux - everything is ok with the Dark Theme. Also, if you look at the patch, I don't really change much, there used to be `container.setBackground(Colors.WHITE);`, and I just changed var name to apply to the same logical container.

> and if "container1" is a good variable name.

I'm all for descriptive var names, but `innerContainer` is a bit too long IMHO, and stuff like `inContainer`, `innerCntnr` is inconsistent/confusing.